tests/runner.js

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
-require( 'babel-core/register' );
+require( 'babel-core/register' )( {
+	ignore: /\/node_modules\/(?!(@automattic|dops-components)\/)/
```

Comment:
Is `node_modules/dops-components` valid? I assumed this would be namespaced. In the original regular expression I'd made, I also included `wp-calypso`, basing this off of the [`resolve.root` from your Webpack configuration](https://github.com/Automattic/jetpack/blob/6881ed6311bda50f41fb1630918f7fd45a35bbf3/webpack.config.js#L74-L76). If `node_modules/wp-calypso/client` is a valid root, it should also be included in the `NODE_PATH` for the test script.